.github/workflows/clean_ami.yml

```diff
@@ -21,5 +22,5 @@ jobs:
           aws-secret-access-key: ${{ secrets.TERRAFORM_AWS_SECRET_ACCESS_KEY }}
           aws-region: us-west-2
 
-      - name: Clean old aws resources
-        run: make integration-cleaner
\ No newline at end of file
+      - name: Clean old ami
+        run: go run ./integration/clean/clean_ami/clean_ami.go --tags=clean
```

Comment:
Part of me thinks that we can rework the make command to allow us to pass in specific tags, and then we could have something like an `ami` tag, and a `dedicatedhost` tag or something and then keep it all wrapped inside the Makefile. Not sure. That was my gut reaction but I don't know how clean that would be.

Reply:
I don't think it worth wrapping the makefile since I moved the cleaners into separate packages.